client/drawtext.lua

```diff
@@ -1,40 +1,26 @@
 local function hideText()
-    SendNUIMessage({
-        action = 'HIDE_TEXT',
-    })
+    lib.hideTextUI()
 end
 
 local function drawText(text, position)
     if type(position) ~= "string" then position = "left" end
-
-    SendNUIMessage({
-        action = 'DRAW_TEXT',
-        data = {
-            text = text,
-            position = position
-        }
+    lib.showTextUI(text, {
+        position = position
     })
 end
```

Comment:
I suggest adding the variable I added here at the top instead and adding `position = positions[position] or position` to the changeText function as well to finish this PR
```suggestion
local positions = {
    left = 'left-center',
    right = 'right-center',
    top = 'top-center'
}

local function drawText(text, position)
    lib.showTextUI(text, {
        position = positions[position] or position
    })
end
```